hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/ApplyMCConfigMessageTask.java

```diff
@@ -25,11 +25,14 @@
 import com.hazelcast.internal.management.ManagementCenterService;
 import com.hazelcast.internal.management.dto.ClientBwListDTO;
 import com.hazelcast.internal.nio.Connection;
+import com.hazelcast.security.permission.ManagementPermission;
 
 import java.security.Permission;
 
 public class ApplyMCConfigMessageTask extends AbstractCallableMessageTask<RequestParameters> {
 
+    public static final Permission REQUIRED_PERMISSION = new ManagementPermission("applyMCConfig");
```

Comment:
Name of the class is a bit confusing but what this method does in reality is applying client black/white listing filters. Since this will be used as an action when configuring permissions for a Management Center client in the member config, we need to make it clearer what this actually does. `clientfiltering.applyConfig` or something similar might be more clear. We can rename the class/methods later in a separate PR.